cli/src/feature.rs

```diff
@@ -104,6 +104,8 @@ impl Ord for CliFeature {
 pub struct CliFeatures {
     pub features: Vec<CliFeature>,
```

Comment:
It would be kind of cool to have these sorted into arrays by status, but I didn't go there since it would break current json output

Reply:
Yeah it would be but I agree it's better keep the existing interface for now